llvm/test/tools/dsymutil/ARM/stmt-seq-macho.test

```diff
@@ -83,14 +83,15 @@ ld64.lld \
 
 # Convert executable to YAML for the test
 echo "#--- stmt_seq_macho.o.yaml"
-obj2yaml stmt_seq_macho.o
+obj2yaml stmt_seq_macho.o | sed '1a IsLittleEndian: true'
```

Comment:
The `sed` invocation here uses a GNU extension: https://www.gnu.org/software/sed/manual/html_node/Other-Commands.html#Other-Commands

```suggestion
obj2yaml stmt_seq_macho.o | sed '1a\
IsLittleEndian: true'
```